Two-step log collection guideline (#27184)
DataDog/documentation: content/en/data_jobs/airflow.md

3. Deploy your updated `requirements.txt` and [Amazon MWAA startup script][3] to your Amazon S3 folder configured for your Amazon MWAA Environment.

> 4. Optionally, set up Log Collection by following the steps below:

Reviewer: ```suggestion
4. Optionally, set up Log Collection for correlation to DAG run executions in DJM by following the steps below:
```

> 4. Optionally, set up Log Collection for correlating task logs to DAG run executions in DJM by following the steps below:

Reviewer 2: ```suggestion
4. Optionally, set up Log Collection for correlating task logs to DAG run executions in DJM:
```

> 4. Optionally, set up Log Collection for correlating task logs to DAG run executions in DJM by following the steps below:
>    1. Configure Amazon MWAA to [send logs to CloudWatch][9].
>    2. [Send the logs to Datadog][10].

Reviewer 2: I could go either way on this suggestion. If this step is confusing and needs to be broken down, leave as is!

```suggestion
4. Optionally, set up Log Collection for correlating task logs to DAG run executions in DJM by configuring Amazon MWAA to [send logs to CloudWatch][9], then [send the logs to Datadog][10].
```

[6]: https://docs.aws.amazon.com/mwaa/latest/userguide/mwaa-create-role.html
[7]: https://app.datadoghq.com/data-jobs/
[8]: https://openlineage.io/docs/integrations/airflow/
[9]: https://docs.aws.amazon.com/mwaa/latest/userguide/monitoring-airflow.html#monitoring-airflow-enable

> [10]: https://docs.datadoghq.com/integrations/amazon_web_services/?tab=roledelegation#log-collection

Reviewer 2: ```suggestion
[10]: /integrations/amazon_web_services/?tab=roledelegation#log-collection
```
